test/powershell/Modules/Microsoft.PowerShell.Utility/Select-String.Tests.ps1

```diff
@@ -71,6 +71,33 @@ Describe "Select-String" -Tags "CI" {
 	    $testinputone | Select-String -Pattern "goodbye" -NotMatch | Should -Be "hello", "hello"
 	}
 
+	it "Should return an array of matching strings with the first match highlighted when Emphasize is used" {
+		if ($IsWindows)
+		{
+			$testinputone | Select-String -Pattern "l" -Emphasize | Should -Be "he*l*lo", "he*l*lo"
+		}
+		else
+		{
+			$testinputone | Select-String -Pattern "l" -Emphasize | Should -Be "he[31ml[0mlo", "he[31ml[0mlo"
```

Comment:
Shouldn't there be an ESC byte before each `[`?
```suggestion
			$testinputone | Select-String -Pattern "l" -Emphasize | Should -Be "he`e[31ml`e[0mlo", "he`e[31ml`e[0mlo"
```

Reply:
Thanks, I added the ESC byte.